meteor/lib/collections/Segments.ts

```diff
@@ -4,16 +4,19 @@ import { IBlueprintSegmentDB } from '@sofie-automation/blueprints-integration'
 import { SegmentNote } from '../api/notes'
 import { createMongoCollection } from './lib'
 import { registerIndex } from '../database'
+import { MongoFieldSpecifier } from '../typings/meteor'
 
 /** A string, identifying a Segment */
 export type SegmentId = ProtectedString<'SegmentId'>
 /** A "Title" in NRK Lingo / "Stories" in ENPS Lingo. */
 
 export enum SegmentOrphanedReason {
 	DELETED = 'deleted',
-	CHANGED = 'changed',
```

Comment:
Was `CHANGED` not used before this update, or why is it removed?

Reply:
Before `unsynced` was renamed/refactored to `orphaned`, this enum was a part of our hack used to distinguish whether segment was unsynced because of being deleted in iNews, or because of being changed while on air. We used that to glue an on air segment to the top of the Playlist if it was removed in iNews, and new segments arrived. With `orphaned` came `syncIngestUpdateToPartInstance`, allowing on-air segments to be updated, so the only way to get orphaned, was by deletion, making this enum unnecessary for a while. But now we actually need another orphaning reason, in order to act accordingly when an on-air segment is hidden.